akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteRestartedQuarantinedSpec.scala

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.remote
+
+import akka.remote.transport.AssociationHandle
+
+import language.postfixOps
+import scala.concurrent.duration._
+import com.typesafe.config.ConfigFactory
+import akka.actor._
+import akka.remote.testconductor.RoleName
+import akka.remote.transport.ThrottlerTransportAdapter.{ ForceDisassociateExplicitly, ForceDisassociate, Direction }
+import akka.remote.testkit.MultiNodeConfig
+import akka.remote.testkit.MultiNodeSpec
+import akka.remote.testkit.STMultiNodeSpec
+import akka.testkit._
+import akka.actor.ActorIdentity
+import akka.remote.testconductor.RoleName
+import akka.actor.Identify
+import scala.concurrent.Await
+
+object RemoteRestartedQuarantinedSpec extends MultiNodeConfig {
+  val first = role("first")
+  val second = role("second")
+
+  commonConfig(debugConfig(on = false).withFallback(
+    ConfigFactory.parseString("""
+      akka.loglevel = WARNING
+      akka.remote.log-remote-lifecycle-events = WARNING
+
+      # Keep it long, we don't want reconnects
+      akka.remote.retry-gate-closed-for  = 1 s
+
+      # Important, otherwise it is very racy to get a non-writing endpoint: the only way to do it if the two nodes
+      # associate to each other at the same time. Setting this on will ensure that the right scenario happens.
+      akka.remote.use-passive-connections = off
```

Comment:
Oh sorry, the comment needs to be fixed
2015.04.22. 20:16 ezt írta ("Patrik Nordwall" <notifications@github.com>):

> In
> akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteRestartedQuarantinedSpec.scala
> <https://github.com/akka/akka/pull/17263#discussion_r28899673>:
>
> > +
> > +object RemoteRestartedQuarantinedSpec extends MultiNodeConfig {
> > +  val first = role("first")
> > +  val second = role("second")
> > +
> > +  commonConfig(debugConfig(on = false).withFallback(
> > +    ConfigFactory.parseString("""
> > +      akka.loglevel = WARNING
> > +      akka.remote.log-remote-lifecycle-events = WARNING
> > +
> > +      # Keep it long, we don't want reconnects
> > +      akka.remote.retry-gate-closed-for  = 1 s
> > +
> > +      # Important, otherwise it is very racy to get a non-writing endpoint: the only way to do it if the two nodes
> > +      # associate to each other at the same time. Setting this on will ensure that the right scenario happens.
> > +      akka.remote.use-passive-connections = off
>
> Perhaps there is a *on* word to many in the comment:
> Setting this *on* will ensure that the right scenario happens
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akka/akka/pull/17263/files#r28899673>.
>